source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,381 @@
+
+### We can now draw our timing diagram with exact parameters:
+
+![](images/tmPostSensor.svg)
+
+We can see that we are sampling the sensors once per control task. That's because our User Event Ratio of `10` fits perfectly with the ratio between the PWM frequency and our control task's frequency. This is the gold standard. In this tutorial we will experiment with changes to the parameters of the Timing Manager and observe the effects on control task timings.
+
+## Experiment 1 - Ratio is too large
+
+Increasing the User Event Ratio is one way to cause tasks to run at less than their desired frequency.
+
+If we increase the User Event Ratio, we can cause the control task to run at less than 10kHz. Let's increase it to 20 by putting `timing_manager_set_ratio(20)` in the `controller_init()` function.
+
+`app_controller.c`:
+```C
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // set User Event Ratio
+    timing_manager_set_ratio(20);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+What does this do? We've made it so that the sensors will collect data every 20 PWM cycles, and then the scheduler will run the control task. Since our PWM frequency is still 10kHz, that means our sensors will collect data every 200us. It also means that our control task can only run once every 200us, instead of 100us like it should.
+
+![](images/tmPostSensorRatio20.svg)
+
+Rebuild and run the new program, and use the command `ctrl stats print` to view the loop time (after doing `ctrl init`).
+
+```
+Task Stats:
+Loop Num:	26806 samples
+Loop Min:	193.78 usec
+Loop Max:	206.22 usec
+Loop Mean:	200.00 usec
+Loop Var:	0.02 usec
+Run Num:	26932 samples
+Run Min:	3.25 usec
+Run Max:	4.23 usec
+Run Mean:	3.41 usec
+Run Var:	0.00 usec
+```
+
+The Loop Mean is how much time there is between successive executions of the control task. It should be `1` / `TASK_CONTROLLER_UPDATES_PER_SEC`, but in this case, it is twice that. That indicates that the control task is only running at half of `TASK_CONTROLLER_UPDATES_PER_SEC`.
+
+Making the User Event Ratio too high is one way that control tasks can be slowed down past their target `TASK_CONTROLLER_UPDATES_PER_SEC`.\
+This violates [inequality number 1](#timing-manager-configuration-rules)
+
+## Experiment 2 - Ratio is too small
+
+By decreasing the User Event Ratio, we can cause multiple sensor samples to occur before one cycle of the control task. This causes a race condition and can lead to inconsistent sensor staleness times.
+
+Let's set the User Event Ratio to `1`.
+
+`app_controller.c`:
+```C
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // set User Event Ratio
+    timing_manager_set_ratio(1);
```

Comment:
Since `EVENT_RATIO` as if it is a define in inequality (1), I suggest you revise your code to have this as a `#define`. Then throughout these experiments, you just have to change this one define value. This will help make everything a bit more consistent with the inequality (easier to understand) and it makes your article a bit shorter (nice).